app/src/main/java/app/olauncher/ui/AppDrawerAdapter.kt

```diff
@@ -195,7 +194,7 @@ class AppDrawerAdapter(
                 }
                 etAppRename.addTextChangedListener(object : TextWatcher {
                     override fun afterTextChanged(s: Editable?) {
-                        etAppRename.hint = getAppName(etAppRename.context, appModel.appPackage)
```

Comment:
I am not sure about this piece of code. It seems to reload the app name every time something is typed. For what is this needed?

The updated code currently does not do this, and always uses the originalLabel set in Utils/getAppsList, which might need to be changed to restore the original behavior.